htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/UrlUtils.java

```diff
@@ -0,0 +1,35 @@
+package io.github.wimdeblauwe.htmx.spring.boot.mvc;
+
+import jakarta.servlet.http.HttpServletRequest;
+
+class UrlUtils {
+
+    /**
+     * Creates a target URL by prepending the context path if {@code contextRelative}
+     * is {@code true} and the URL starts with a slash ("/").
+     *
+     * @param request
+     * @param url
+     * @param contextRelative
+     * @return
+     */
+    static String createTargetUrl(HttpServletRequest request, String url, boolean contextRelative) {
+        if (contextRelative && url.startsWith("/")) {
+            // Do not apply context path to relative URLs.
```

Comment:
I was a bit confused by this comment. But that comment is there to explain the `url.startsWith("/")` I think?

Maybe you can drop the comment by adding a method `isRelativeUrl(url)` and use `if(contextRelative && !isRelativeUrl(url))` ?

Reply:
That is correct. I copied it from the "RedirectView", but surely I can remove it.

Reply:
Removed the comment.